add_submission_offsets.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import print_function
+
+from flask.cli import FlaskGroup
+import click
+from collections import defaultdict
+
+import db
+import webserver
+
+from sqlalchemy import text
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app_flaskgroup)
+
+
+@cli.command(name='add-offsets')
+@click.option("--limit", "-l", default=10000)
+def add_offsets(limit):
+    """Update lowlevel submission offsets with a specified limit."""
+    incremental_add_offset(limit)
+
+
+def incremental_add_offset(limit):
+    with db.engine.connect() as connection:
+
+        # Find number of items in table
+        size_query = text("""
+            SELECT count(*) AS size
```

Comment:
we're missing some rows in this table, so max(id) is not the same as the number of rows, I changed it